packages/react-redux/src/slices/cartoSlice.d.ts

```diff
@@ -149,3 +149,5 @@ export function setDrawingToolEnabled(enabled: boolean): {
 };
 
 export function selectSpatialFilter(state: any, sourceId?: string): object | null;
+
+export function selectDrawingToolMode(state: any): string | false;
```

Comment:
If 'false' is equivalent to 'None', I think it could be better to add that option to an enum or type

Reply:
`selectDrawingToolMode` is a selector that return `false` if the `spatialFilter` is null in the store (the default state). I think that creating another enum type could be over-engineering

Reply:
I just see there are other options without adding boolean to the mix:
- a) just an enum / type: DrawingToolMode.Type1, DrawingToolMode.Type2,... + DrawingToolMode.None, or more simple
- b) just respecting the null for 'None'. 

"selectDrawingToolMode" doesn't sound like a boolean to me, but a value from a predefined list. If required, we can later do in code something like this

```
const noFilter = selectDrawingToolMode === null
if (noFilter) ...
```

What do you think?

Reply:
I've done the `b` option